docs/sagemaker.md

```diff
@@ -27,9 +27,9 @@ Here's a list of frameworks and versions which support this experience.
 
 | Framework | Version |
 | --- | --- |
-| [TensorFlow](tensorflow.md) | 1.15 |
+| [TensorFlow](tensorflow.md) | 1.15, 2.1 |
 | [MXNet](mxnet.md) | 1.6 |
-| [PyTorch](pytorch.md) | 1.3 |
+| [PyTorch](pytorch.md) | 1.3, 1.4, 1.5 |
```

Comment:
Can you confirm that PyTorch 1.3 supported for ZCC for sure?
I think DLCs now support Debugger in PyTorch >= 1.4.
I asked/confirmed it with Kiran.